### PR TITLE
Add: Company proto, new field to SmsMessage.proto: SmsMessage.Company

### PR DESCRIPTION
**Pull request by cotom972**
1. Create a new `Company.proto`:
```
    string name = 1;
    string id = 2;
    string address = 3;
    string phone = 4;
    string fax = 5;
```

2. Add it as a field to `SmsMessage.proto`:
  ```
  message SmsMessage {
    ...
    ...

    Company company = 7;
```

Change lawsuit form (&LawsuitProto) to work with the company proto - instead of individual company fields.

### CODE REVIEW

File: projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/MainActivity.java [hunk 1]
```diff
@@ -78,18 +78,6 @@ protected void onCreate(Bundle savedInstanceState) {
...
-     * Intersect spams/dbMessages with user's inbox messages, display only the relevant messages
-     * that are in the inbox.
-     *
-     * <p>// Create a list of the intersection between the two lists, messages and spam // Based on
-     * https://www.baeldung.com/java-lists-intersection List<SmsMessage> spamAndInbox =
-     * inbox.stream().distinct().filter(spams::contains).collect(Collectors.toList());
-     * List<SmsMessage> suggestionsAndInbox =
-     * inbox.stream().distinct().filter(dbMessages::contains).collect(Collectors.toList());
-     */
-
```
Comment (cotom972) [inline projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/MainActivity.java]: Done in a previous PR
https://github.com/hasadna/monorepo/pull/256

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/LawsuitActivity.java [hunk 2]
```diff
@@ -328,11 +329,11 @@ private String fillTemplate() throws IOException {
     lawsuit = lawsuit.replace("<userPhone>", lawsuitProto.getUserPhone());
 
     // Company
-    lawsuit = lawsuit.replace("<companyName>", lawsuitProto.getCompanyName());
-    lawsuit = lawsuit.replace("<companyId>", lawsuitProto.getCompanyId());
-    lawsuit = lawsuit.replace("<companyAddress>", lawsuitProto.getCompanyAddress());
-    lawsuit = lawsuit.replace("<companyPhone>", lawsuitProto.getCompanyPhone());
-    lawsuit = lawsuit.replace("<companyFax>", lawsuitProto.getCompanyFax());
+    lawsuit = lawsuit.replace("<companyName>", lawsuitProto.getCompany().getName());
+    lawsuit = lawsuit.replace("<companyId>", lawsuitProto.getCompany().getFax());
```
Comment (oferb) [inline projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/LawsuitActivity.java]: This should be `getId()` probably.

File: projects/noloan/common/src/main/proto/Sms.proto [hunk 3]
```diff
@@ -11,9 +13,11 @@ message SmsMessage {
     string receivedAt = 3;
     string id = 4;
     bool approved = 6;
+    Company company = 7;
```
Comment (oferb) [inline projects/noloan/common/src/main/proto/Sms.proto]: We don't renumber proto fields that are stored in binary form. `SmsMessage` is stored in binary form in the DB.
If you do that, you will get weird errors.

So please leave the field numbers as they are.

File: projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/LawsuitActivity.java (hunk 2 above)
Comment (cotom972) [reply projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/LawsuitActivity.java]: :+1:

File: projects/noloan/common/src/main/proto/Sms.proto (hunk 3 above)
Comment (cotom972) [reply projects/noloan/common/src/main/proto/Sms.proto]: Didn't think of that.... :+1:

Comment (cotom972) [reply projects/noloan/common/src/main/proto/Sms.proto]: I deleted the messages in the DB - Starting over with this ordering.

Pushed it as it is...

**Pull request merged**